apps/sel4test-tests/src/tests/scheduler.c

```diff
@@ -1624,8 +1624,10 @@ static int test_simple_preempt(struct env *env)
 
     return sel4test_get_result();
 }
+/* This test is flaky under simulation. Probably a race condition that only
+   comes out under simulator timing conditions. See also #42  */
```

Comment:
Same here, better say "GitHub Issue #42" to give a bit more context that this number refers to.